Compiler/src/Compiler.cpp

```diff
@@ -3703,13 +3704,24 @@ void compileOrThrow(BytecodeBuilder& bytecode, AstStatBlock* root, const AstName
 
     Compiler compiler(bytecode, options);
 
-    // since access to some global objects may result in values that change over time, we block table imports
-    for (const char* global : kSpecialGlobals)
+    // since access to some global objects may result in values that change over time, we block imports from non-readonly tables
+    if (FFlag::LuauGenericSpecialGlobals)
     {
-        AstName name = names.get(global);
+        if (AstName name = names.get("_G"); name.value)
+            compiler.globals[name].writable = true;
 
-        if (name.value)
-            compiler.globals[name].special = true;
+        if (options.mutableGlobals)
+            for (const char** ptr = options.mutableGlobals; *ptr != NULL; ++ptr)
+            {
+                if (AstName name = names.get(*ptr); name.value)
+                    compiler.globals[name].writable = true;
+            }
+    } else {
```

Comment:
k&r braces

Reply:
I don't know what that means, I did a code search in Luau and this style is used all the time

Reply:
Yeah the change looks good after your fix. The brace style you had before is called "K&R" -- we never use it in Luau.